Update how-to-set-up-a-graphical-interface.md (#130)
canonical/open-documentation-academy: multipass/how-to/how-to-set-up-a-graphical-interface.md

The images used by Multipass do not come with a graphical desktop installed. For this reason, a desktop environment must be installed (here we use `ubuntu-desktop` but there are as many other options as flavors of Ubuntu exist), along with the RDP server (we will use `xrdp` but there are also other options such as `freerdp`). 

> To do this, first you need to choose a Multipass instance. List your instances:

Reviewer: ```suggestion
To do this, you need to log into a running Multipass instance. Start by listing your instances:
```

Reviewer: I'm suggesting a slight rephrasing here.

```plain
multipass list
```

Sample output:

```plain
Name                    State             IPv4             Image
headbanging-squid       Running           10.49.93.209     Ubuntu 22.04 LTS
```

> Next log in to the running Multipass instance:

Reviewer: ```suggestion
Next open a shell into the running instance:
```

```plain
multipass shell headbanging-squid
```

Once inside the instance, run the following commands to install `ubuntu-desktop` and `xrdp`:

```plain

> sudo apt update

Reviewer: Thanks for spotting these extra `$` signs! They haven't been cleaned up consistently from all pages yet, this is a good contribution while you're editing this page :)

Author: my pleasure, once I have the right Trust level maybe I can script it. It would be easier to do locally in a GitHub fork, but I assume that it is a lot of work to sync back and forth between discourse and Git.

Reviewer: > my pleasure, once I have the right Trust level maybe I can script it. It would be easier to do locally in a GitHub fork, but I assume that it is a lot of work to sync back and forth between discourse and Git.

Yes, indeed... for the time being, it's mostly a manual process as we don't see the need - for now - to come up with a semi-automated solution. Still, if you spot style guide inconsistencies on a number of pages, let me know and we can process them in batches, if you like :)

sudo apt install ubuntu-desktop xrdp
```

Then, we need a user with a password in order to log in. One possibility is setting a password for the default `ubuntu` user:

```plain
sudo passwd ubuntu
```

You will be asked to enter and re-enter a password. 

You are done on the server side!

> Now, quit the Ubuntu shell on the instance with the `logout` command and find out in the host the IP address to connect to. The IP address is in the output of `multipass list` from the first step:

Reviewer: ```suggestion
Now, quit the Ubuntu shell on the running instance with the `exit` command, and take note of the IP address to connect to. The instance's IP address can be found in the output of `multipass list` from the first step above, or you can use the `multipass info` command as well.
```

Reviewer: Since we've already displayed the IP address in the `multipass list` output, there's no need to re-run it. I think it's worth mentioning the `multipass info` command as well, just as a reminder.

```plain

> $ multipass list

Reviewer: ```suggestion
multipass info headbanging-squid
```

Reviewer: If we choose to suggest using this command, the sample output below should also be updated accordingly.

Author: I will run the list and add the output.

```plain
multipass list
```

Sample output:

```plain
Name                    State             IPv4             Image
headbanging-squid       Running           10.49.93.209     Ubuntu 22.04 LTS
```

Next open a shell into the running instance:

```plain
multipass shell headbanging-squid
```

Once inside the instance, run the following commands to install `ubuntu-desktop` and `xrdp`:

```plain
sudo apt update
sudo apt install ubuntu-desktop xrdp
```

Then, we need a user with a password in order to log in. One possibility is setting a password for the default `ubuntu` user:

```plain
sudo passwd ubuntu
```

You will be asked to enter and re-enter a password. 

You are done on the server side!

Now, quit the Ubuntu shell on the running instance with the `exit` command, and take note of the IP address to connect to. The instance's IP address can be found in the output of `multipass list` from the first step above, or you can use the `multipass info` command as well.

```plain
multipass info headbanging-squid
```

Sample output:

```plain
Name:           headbanging-squid
State:          Running
Snapshots:      0
IPv4:           10.49.93.209

> Release:        Ubuntu 24.04 LTS

Reviewer: Here as well: 

```suggestion
Release:        Ubuntu 22.04 LTS
```

Reviewer: @DanRoscigno check the LTS version used in the example, it should be 22.04 for `headbanging-squid`

> Image hash:     2e0c90562af1 (Ubuntu 24.04 LTS)

Reviewer: In our example, the image used is Ubuntu 22.04 LTS, so I'll fix that

```suggestion
Image hash:     2e0c90562af1 (Ubuntu 22.04 LTS)
```
